spec/support/shared/spree_builders.rb

```diff
@@ -1,10 +1,14 @@
 RSpec.shared_context 'spree_builders' do
   def build_spree_product(id: 1, name: 'name', description: 'description',
+                          master: build_spree_variant,
+                          title: 'title',
                           vendor: 'vendor', slug: 'slug',
                           pos_product_id: '123',
                           created_at: build_date_time,
                           updated_at: build_date_time,
-                          available_on: build_date_time)
+                          available_on: build_date_time,
```

Comment:
I would agree that this is getting out of hand quick. But this really helps in term of the speed of the unit test. This allow us to inject stuff easily. I am unsure how else to proceed, this was suggested by DW and I thought I would give it a whirl. 

Do you have suggestions?